angularjs-portal-home/src/main/webapp/my-app/marketplace/partials/marketplace.html

```diff
@@ -3,67 +3,74 @@
   <app-header app-title="Browse {{NAMES.title}}"
               app-icon="fa-home">
   </app-header>
-  <div class="mp-header">
-    <input type="text" class="marketplace-search" placeholder="What are you looking for?" ng-model="searchText" select-on-page-load aria-label="Search bar enter the app you are looking for">
-  </div>
-  <div class="mp-opaque">
-    <ul>
-      <li ng-click="selectFilter('popular','')" ng-class="{true:'selected-filter'}[selectedFilter === 'popular']"><a href="#" aria-label="search from the most popular apps">Most Popular</a></li>
-      <li ng-click="selectFilter('az','')" ng-class="{true: 'selected-filter'}[selectedFilter === 'az']"><a href="#" aria-label="search by alphabetical order">A-Z</a></li>
-      <li ng-click="selectFilter('category','')" ng-class="{true: 'selected-filter'}[selectedFilter === 'category']"><a href="#" aria-label="search by categories">Categories</a></li>
-    </ul>
-  </div>
-  <div class="show-all-div">
-    <p><span ng-if="portlets.length > 0">Showing {{ (portlets | filter:searchTermFilter | showApplicable:showAll | showCategory:categoryToShow).length }} of {{ portlets.length }} results in {{NAMES.title}}.</span>
-    <span ng-if="portlets.length == 0">Loading...</span></p>
-    <div ng-hide="GuestMode"><p><input type="checkbox" ng-click="toggleShowAll()" ng-checked="showAll">Show all<span popover="Show all apps, even if I don't have access" popover-append-to-body="true" popover-placement="left" popover-trigger="mouseenter"><i class="fa fa-question-circle" ></i></span></p></input></div>
-  </div>
-  <div ng-show="showCategories" class="show-categories">
-    <p>Categories:</p><a ng-repeat="category in categories" href="" class="category-links" ng-click="selectFilter('category',category)" ng-class="{true: 'selected-category'}[categoryToShow === category]">{{category}}</a>
-  </div>
+  <div class="marketplace-body">
+    <div class="mp-header">
+      <div class="browse-help">
+        <p class="hidden-xs">What are you looking for?</p>
+        <p class="visible-xs"><i class="fa fa-search"></i></p>
+      </div>
+      <div class="browse-input">
+        <input type="text" class="hidden-xs marketplace-search" placeholder="e.g. Benefits Information" ng-model="searchText" select-on-page-load aria-label="Search bar enter the app you are looking for">
```

Comment:
I think the randomly selected app could over-complicate the intention of a simple placeholder. Being generic like Courses is a more palatable approach. Someday it would be grant to start implementing ngTranslate (think property files in the form of json mixed with directives and filters).